<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendEmail</class>
 <widget class="QDialog" name="SendEmail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>884</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>841</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>Address</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>From:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>To:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Cc:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Bcc:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editSendEmailFromComm">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>721</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="editSendEmailCommTo">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>721</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="editSendEmailCommCc">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>721</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="editSendEmailCommBcc">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>721</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>230</y>
     <width>120</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Send Email</string>
   </property>
   <widget class="QPushButton" name="btnSendEmailCommSend">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>691</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Subject Line</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>25</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Subject:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editSendEmailCommSubject">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>25</y>
      <width>571</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Attachments</string>
   </property>
   <widget class="QPushButton" name="btnSendEmailCommAttach">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>61</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>841</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>Message</string>
   </property>
   <widget class="QLineEdit" name="editSendEmailCommMessage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>801</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnSendEmailCommClose">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>590</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
